### ISSUE TITLE
There is a little problem about the $$\label{}$$

### ISSUE THREAD

**Issue by Fenyutanchan** [typora/typora-issues#3043]
Hey, I have found a strange problem. I wrote some math equations in the Math Block, with \label{test} to label the equation and refer it in the context, and it worked really well. However, if I save it and reopen this .md file again, the strange thing happens. Almost every Math Block with "\label{}" cannot be displayed normally, and there are the same warnings says, "Label 'test' multiply defined". But if I open a new .md file, and put these source codes to the new file, they will work well again. I think there are some cache files disturbing the Math Block interpreter, can you help me?
![KT(J56VL_T9TAWBE( 8SDV](https://user-images.githubusercontent.com/32387646/69488193-fa975780-0ea0-11ea-9540-aee2e0df03c0.png)

**Comment by abnerlee** [comment]
What's your Typora version? Possible to send us <hi@typora.io> a sample md file?

**Comment by Fenyutanchan** [comment]
> What's your Typora version? Possible to send us [hi@typora.io](mailto:hi@typora.io) a sample md file?

Version: 0.9.79(beta)
Platform: Microsoft Windows [版本 10.0.18362.476]

The source codes I have sent it to you by [fenyutanchan@vip.qq.com](mailto:fenyutanchan@vip.qq.com)

**Comment by Liguobing00** [comment]
I have the same problem for quite a long time. I have checked all the versions and only the versions before 0.9.70 don't have this kind of problem. I think it is because Typora has a big update and uses Electron 5.0 after that. My platform is Linux Mint (18.3). My solving method is to use the old version (0.9.70) of Typora. This problem keeps me from using the latest version and I hope it will be fixed soon.

**Comment by Liguobing00** [comment]
The following is another similar bug as discussed in this issue.
![Selection_086](https://user-images.githubusercontent.com/14992913/76301719-b027d280-62f9-11ea-9105-2ed34cd0718c.png)

**Comment by physxz** [comment]
Same problem
Version: 0.9.86(beta)
Platform: Microsoft Windows [Version 10.0.18362.720]

**Comment by KyQiao** [comment]
Same problem here
Version: 0.9.86(beta)
The md file renders fine with a standalone mathjax.

**Comment by huang-ying** [comment]
Same problem here with 0.9.86 (beta). Windows 10

**Comment by Guillaume-Garrigos** [comment]
Same problem with 0.9.65. W10

**Comment by shawn233** [comment]
Same problem. Version: 0.9.97. Windows 10

**Comment by abnerlee** [comment]
relates #3455

**Comment by shawn233** [comment]
> Same problem. Version: 0.9.97. Windows 10

My problem is fixed by checking the "Settings->Markdown->Auto Numbering Math Equations" option. Hope this helps.

**Comment by chinsengi** [comment]
Same problem version 0.9.98, @shawn233 I don't want auto numbering because I add an invisible math block at the front to include all of my macros, autonumbering will put a number on that block, which makes it look silly :)

**Comment by abnerlee** [comment]
same with #4195

**Comment by JackyToU** [comment]
> relates #4195
> 
> should be fixed
still in the latest release 0.11.8 
![image](https://user-images.githubusercontent.com/2480916/132994554-9e3e0820-57a2-4200-8ab9-46edf89ea6c3.png)
![1631462472(1)](https://user-images.githubusercontent.com/2480916/132994575-a9c2bef4-0f85-47cf-8b85-57805e8abcfd.png)

**Comment by adamblvck** [comment]
Working on the latest version of Typora 1.6.7, and am getting the same error, and the reference system is not working.

<img width="537" alt="Screenshot 2023-07-17 at 02 32 47" src="https://github.com/typora/typora-issues/assets/22320876/825f1d7b-217f-489a-b9e2-af56a6be6ff4">

**Comment by kimbostroem** [comment]
Why is this issue marked as closed? It still persists!
The bug still exists in version 1.7.6 as of Nov 2023. And it does not go away when checking "Auto Numbering Math Equations". It is healed (not fixed) when pressing Edit -> Math Tools -> Refresh All Math Expressions.

**Comment by kimbostroem** [comment]
[Muskeloptimierung.md](https://github.com/typora/typora-issues/files/13389709/Muskeloptimierung.md)

**Comment by Zwchen1** [comment]
still has the problem...Nobody fix it...
